test/parallel/test-stream-writable-clear-buffer.js

```diff
@@ -1,33 +1,32 @@
 'use strict';
-const common = require('../common');
+require('../common');
 const Stream = require('stream');
 // This test ensures that the _writeableState.bufferedRequestCount and
-// the actual buffered request count are the same
+// the actual buffered request count are the same.
```

Comment:
Perhaps, while we are at it, move the comment under or above `require`s as IMO it looks strange for an explanation to the test to be in the middle of  `require`s.

Reply:
I agree. Moved. Thanks.